### ISSUE TITLE
Preparation for major URL exposure

### ISSUE THREAD

**Issue by Hasmig** [GenEdProject/TeachGenocideSite#141]
April 1-30, "GenocideEducation.org" will be printed on a large banner mounted on Treasure Island, in view of Bay Bridge drivers. I'd like to verify that our server can sustain a possibly large increase in traffic during that period, as well as attempts at hacking/breaking the site.

**Comment by copacetic** [comment]
Yes, it will definitely be able to handle it. I can also turn on a caching mode that will make it really fast, but our updates will take longer to become user facing, at that point

**Comment by Hasmig** [comment]
What does caching mode mean in this context and what will it make really fast?  
Can you show us how to check the metrics for our site (number of visitors, pages visiting, length of visits, etc...)?

**Comment by copacetic** [comment]
I can try getting this done tonight. I'll need access to the Google account
though.

**Comment by Hasmig** [comment]
The email address Seda created (I guess its just for Googl's admin purposes) is TheGenocideEducationProject@gmail.com  PW: gened541
